### ISSUE TITLE
Biome Concept: Dead Forest

### ISSUE THREAD

**Issue by Draylar** [fabric-community/spooky-time#98]
Welcome to... the Dead Forest. Something seems to have swept through the area, bringing a dead silence and eerie atmosphere. Despite walking straight in, you haven't seen any mobs... but you feel like something is watching you.

**Visuals:**
The Dead Forest is comprised mostly of Dark Oak trees; the ground is covered in bush, mushrooms, vines, and fallen trees. A dead trunk could look similar to this:

![2019-10-04_13 26 31](https://user-images.githubusercontent.com/17728338/66247414-e4abb880-e6e1-11e9-9a47-023240a50986.png)

Grown trees would be taller and would have large circle dark oak leaves to cover the top. It's somewhat similar to the [Twilight Forest Dark Forest](http://benimatic.com/tfwiki/index.php?title=Dark_Forest) (may need a better biome name idea), but this version has better visibility and the occasional view of the sky. Trees are more scattered as well.

**Entities:**
The Dead Forest is unique in that there aren't as many easily-visible hostile mobs. One neutral creature you'll stumble on is the _Tree Spirit_, which is a lesser (and smaller) version of the boss. They will occasionally spawn on their own in a passive form, but can also spawn as a result of cutting down dark oak trees near the bottom, in which case they are aggressive towards you. You can also find Termites (name to be changed), which are smaller brown silverfish that hide inside logs and shrubbery.

**Misc:**
Glowing Mushrooms, which seem to thrive in this decaying environment, can be found scattered throughout the land. They emit a medium amount of light and slowly pulse on and off. They can be combined with red and brown mushrooms to make shiny mushroom stew, which gives a bonus regeneration effect. 

Magical Mushrooms, which hide underneath deep shrubbery (and don't like to reveal themselves until forced to), can be used to make the Nature's Essence potion, which grants several useful buffs at once.

**Boss:**
At the center of the Dark Forest, you'll find the cause of the fallen forest: the _Ancient Tree Spirit_, who is basically just a giant walking dark-oak tree. He spawns in large circular clearings surrounded entirely by dead trunks, and the ground in the circle is composed of coarse dirt, gravel, and cobblestone.

**Items:**
- Dark Oak tree leaves will drop _Mossy Dark Saplings_ instead of regular saplings, which spawn trees similar to the image above (except with leaves). 
- The Ancient Tree Spirit drops _Nature's Wand_, which has an undedicated attack/utility mechanic (maybe a projectile launcher?). "_It seems to be rotting, but it's still mostly functional._

**Comment by Shnupbups** [comment]
I think it should be a brand new type of tree instead of Dark Oak. The Dark Oaks in the Spooky dim feel like a placeholder to me.

**Comment by Draylar** [comment]
I should clarify and say they wouldn't be the same Dark Oak trees vanilla uses, just custom trees that use Dark Oak as the log.

**Comment by Shnupbups** [comment]
Yeah, I just don't like the idea of using a vanilla log and think we should make our own wood type. But eh.
